packages/server/dataloader/RedisCache.ts

```diff
@@ -51,7 +51,7 @@ export default class RedisCache<T extends keyof CacheType> {
   // }
   private getRedis() {
     if (!this.redis) {
-      this.redis = new Redis(process.env.REDIS_URL, {connectionName: 'redisCache'})
+      this.redis = new Redis(6379, process.env.REDIS_URL!, {connectionName: 'redisCache'})
```

Comment:
I had to upgrade Redis to be able to sort the sorted set in reverse order by timestamp. After upgrading: 

- the typings have improved which caught a few potential bugs
- the `Redis.Redis` type became `Redis`
- `new Redis()` requires a port number as its first arg. We could add this to the env vars, but I don't think it needs to be kept hidden so I added it here. Alternatively, I could add the number to `constEnums`, for example, and we could always import it from there